types/address/codec.go

```diff
@@ -0,0 +1,25 @@
+package address
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+type Codec interface {
+	// AddressStringToBytes decodes text to bytes
+	AddressStringToBytes(text string) ([]byte, error)
+	// AddressBytesToString encodes bytes to text
+	AddressBytesToString(bz []byte) (string, error)
+}
+
+// VerifyFormat checks validity of address format
```

Comment:
Should we call this `DefaultVerifyFormat`? I ask because this is only called if an application doesn't implement it's own address verifier?